pubsub/sample.rb

```diff
@@ -106,3 +106,89 @@ def list_subscriptions
   end
 end
 # [END list_subscriptions]
+
+# [START print_topic_policy]
```

Comment:
Ah, you're right!  I forgot these were live 😄 

Yeah, let's go ahead and change them.  It looks really nice without the method definition.  Eg. the way Python samples show up on that page